src/test/java/com/google/sps/RankTest.java

```diff
@@ -242,4 +336,23 @@ public void mixedPreferencesDoubleTie() {
     List<Schedule> expected = Arrays.asList(schedule3, schedule4);
     Assert.assertEquals(expected, actual);
   }
+
+  // TODO: Add all the tests for the big schedules, once merged into HandleRecitations branch
+
+  @Test
+  public void bigTestEmptyPreference() {
+    Preferences preferenceList = new Preferences(Arrays.asList());
+    //List<Course> courses = createBigScheduleCourses();
+
+    List<Schedule> schedules = Arrays.asList(bigSchedule1, bigSchedule2, bigSchedule3, bigSchedule4);
+    preferenceList.sortSchedules(schedules);
+
+    List<Schedule> actual = schedules;
+    List<Schedule> expected = Arrays.asList(bigSchedule1, bigSchedule4, bigSchedule3, bigSchedule2);
```

Comment:
Is there a reason we expect this specific sort order when there are no preferences? Should the list order end up getting changed at all when there are no preferences?

Reply:
You're right, the list order should not change when there are no preferences set. I couldn't run tests anyways since it wont compile, until its accessible from `HandleRecitations` branch, so once its merged there, I will correct/add the rest of the tests.